flow360/component/flow360_params/volume_zones.py

```diff
@@ -51,7 +51,6 @@ class ReferenceFrameDynamic(Flow360BaseModel):
 
     center: LengthType.Point = pd.Field(alias="centerOfRotation")
     axis: Axis = pd.Field(alias="axisOfRotation")
-    is_dynamic: bool = pd.Field(True, alias="isDynamic", const=True)
```

Comment:
are you sure we don't need this? How the solver knows that no omega is required here?

Reply:
According to the logic in the solver side preprocessing, we add 0 for theta, omega and omegaDot if they are not specified.

Reply:
the use of "isDynamic" has been deprecated for quite some time.